kani-compiler/src/codegen_cprover_gotoc/utils/names.rs

```diff
@@ -52,34 +50,11 @@ impl<'tcx> GotocCtx<'tcx> {
         format!("{var_name}_init")
     }
 
-    /// A human readable name in Rust for reference, should not be used as a key.
-    pub fn readable_instance_name(&self, instance: Instance<'tcx>) -> String {
-        with_no_trimmed_paths!(self.tcx.def_path_str_with_args(instance.def_id(), instance.args))
-    }
-
-    /// The actual function name used in the symbol table
-    pub fn symbol_name(&self, instance: Instance<'tcx>) -> String {
-        let llvm_mangled = self.tcx.symbol_name(instance).name.to_string();
-        debug!(
-            "finding function name for instance: {}, debug: {:?}, name: {}, symbol: {}",
-            instance,
-            instance,
-            self.readable_instance_name(instance),
-            llvm_mangled,
-        );
-
-        let pretty = self.readable_instance_name(instance);
-
-        // Make main function a special case in order to support `--function main`
-        // TODO: Get rid of this: https://github.com/model-checking/kani/issues/2129
```

Comment:
Can #2129 be closed then?

Reply:
not yet... we have the same comment on the stable version. The problem here is with the communication between kani-driver and the compiler.